### DISCUSSION TITLE
Payments

### DISCUSSION THREAD

**Discussion by intelimangd** [pH7Software/pH7-Social-Dating-CMS#918] [Q&A]
As far as i can tell the option is set once. With a single currency symbol.

VAT is different in different countries.  But seems set in a single place.
Can VAT be set per country?
I would have expected payment gateway providers ti be enabled/disabled per country/currency.

Is my understanding correct?

**Comment by hobb7t** [comment]
Well, for us in EU, VAT is different on each country and this is over buying items from other countries, since June the laws changed and you have to pay the VAT of the country that the item's final destination will be. Now the thing with digital services is the VAT that most probably YOU pay for the money you get from that people, right? Because they are buying digital subscriptions, this is something that we should check. If you want, join the pH7 discord server so we can keep each other updated about it https://discord.gg/p59BhR3AHe
